hpc_config.yaml

```diff
@@ -0,0 +1,27 @@
+# Template for HPC config for run_sim.py
+# None-number values should be quoted 
+conda:
+  # Shorten path will be resolved
+  conda_path: "~/miniconda3"
+  environment_name: "h4c"
+module:
+  # List environment modules to load
+  - "openmpi"
+  - "cuda"
+slurm:
+  # SLURM SBATCH options, listed as --flag: value
+  # Flags not shown here should work
+  partition: "Main"  # Name of partition
+  nodes: 1  # Number of nodes
+  mem: "8GB"  # Memory with unit
+  ntasks: 1  # Number of tasks
+  cpus-per-task: 2  # Number of CPUs per task
+  time: "0-00:30:00"  # slurm-aware time string format
+  gpus: 1  # Number of GPU
+  gres: "gpu:1"  # Some cluster do not support --gpus flag
+  # Job name, will be modified to <sky_model>-fch<freq_chan>-chunk<chunk> 
+  # when running H4C sim via run_sim.py
+  job-name: "hera-sim"
```

Comment:
why have the job name it's just overwritten?

Reply:
for custom sims. there is a cli function in the main script that has not been written. the idea there is to allow custom simulation where the user must specify everything. the job-name should be customizable for that